examples/google-secure-signals-integration/with_sdk_v3/views/intro.html

```diff
@@ -1,12 +1,12 @@
 <h1>Server-Side Integration Example, UID2 JavaScript SDK</h1>
 <p class="intro">
   This example demonstrates how a content publisher can use the UID2 services and the
-  <a href="https://unifiedid.com/docs/sdks/client-side-identity"
-    >Client-Side Identity JavaScript SDK</a
+  <a href="https://unifiedid.com/docs/sdks/sdk-ref-javascript"
+    >UID2 SDK for JavaScript</a
```

Comment:
Suggest we do the same thing and explain:

> This includes:
> 
> 1.     Generate tokens on the server side using v2/token/generate endpoint
> 2.    Pass the token value back to the UID2 SDK For Javascript embedded in the client web page and make it available to be retrieved by the Google Publisher Tag (GPT) javascripts.

Reply:
To be addressed in a later PR, per discussion.